idd/Energy+.idd.in

```diff
@@ -64816,50 +65000,43 @@ WaterHeater:HeatPump,
        \object-list ZoneNames
        \note Used only if Inlet Air Configuration is ZoneAirOnly or ZoneAndOutdoorAir.
        \note Otherwise, leave field blank.
-  A14, \field Tank Object Type
```

Comment:
@mjwitte I see your points here. Especially the consistency one. I will change it back to require object type in all these places. Sigh. 

For the future, we really should have some developer documentation for object design. There were several things that would have saved me time with here.